action/main.py

```diff
@@ -14,9 +14,10 @@
 
 run([sys.executable, "-m", "venv", str(ENV_PATH)], check=True)
 
-req = "black[colorama]"
-if VERSION:
-    req += f"=={VERSION}"
+version_specifier = VERSION
+if VERSION and VERSION[0] in "0123456789":
```

Comment:
```suggestion
if VERSION and VERSION[0].isdigit():
```

Reply:
Ah damn, I didn't notice this comment before I made mine :smile: I can change it, though I would like to note that isdecimal is less wide check than isdigit so it might be better choice if we're switching it.